README.md

```diff
@@ -18,17 +18,17 @@ This documentation is for developers interested in using this Node.js client to
 ## Installation
 
 ```shell
-npm install --save notifications-node-client
+npm install --save @govau-platforms/notify-client
 ```
 
 ## Getting started
 
 ```javascript
-var NotifyClient = require('notifications-node-client').NotifyClient,
-	notifyClient = new NotifyClient(apiKey);
+const { Client } = require('@govau-platforms/notify-client');
+const notifyClient = new Client(apiKey);
```

Comment:
Is this the constructor or is it an object now?

Reply:
`notifyClient` is the object if that's what you're asking